packages/react-noop-renderer/src/createReactNoop.js

```diff
@@ -80,6 +80,7 @@ type CreateRootOptions = {
   unstable_transitionCallbacks?: TransitionTracingCallbacks,
   onUncaughtError?: (error: mixed, errorInfo: {componentStack: string}) => void,
   onCaughtError?: (error: mixed, errorInfo: {componentStack: string}) => void,
+  onDefaultTransitionIndicator?: () => () => void,
```

Comment:
Should the destroy be optional?

Typically you probably need it but if you use optimistic state then you don’t.

Reply:
Made it optional